### PR TITLE
add mock registry data

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by eriknelson** [comment]
ACK

File: extras/mock-registry-data.yml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+      imageUrl: "https://s3.amazonaws.com/fusor/2017demo/NetworkingContentDelivery_AmazonCloudFront.png"
+      tags:
+        - Amazon
+        - Networking
+        - Content Delivery
+      documentationUrl: "https://aws.amazon.com/cloudfront/"
+  - id: 650fc7c3-12e3-4cce-b36c-2984e076768d
+    name: elasticache-apb
+    image: ansibleplaybookbundle/elasticache-apb
+    description: DFully managed, in-memory data store and cache in the cloud. Compatible with Redis and Memcached.  Provides sub-millisecond latency to power real-time applications.
```
Comment (cfchase) [inline extras/mock-registry-data.yml]: DFully -> Fully

File: extras/mock-registry-data.yml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    bindable: true
+    async: optional
+    parameters: {}
+    metadata:
+      displayName: "Amazon Elasticache"
```
Comment (cfchase) [inline extras/mock-registry-data.yml]: Capital C?

Comment (jmontleon) [inline extras/mock-registry-data.yml]: I don't know. In the spreadsheet it's lowercase.

Comment (jmrodri) [inline extras/mock-registry-data.yml]: Official product name is Amazon ElastiCache, so +1 to capital C.

**Comment by jmrodri** [comment]
Is this still relevant?

**Pull request closed without merging**